test/e2e/reconciler/cluster/controller_test.go

```diff
@@ -214,10 +196,13 @@ func TestClusterController(t *testing.T) {
 
 			t.Log("Installing sink cluster...")
 			start := time.Now()
-			_, err = framework.CreateClusterAndWait(t, ctx, source.Artifact, sourceKcpClusterClient.Cluster(wsClusterName), sink)
+			workloadCluster, err := framework.CreateReadyCluster(
```

Comment:
would prefer we stop calling WorkloadCluster Cluster in a number of places.

Reply:
If you intend this comment to be actionable, an alternative name would be helpful.